test/xcode_config_test.bzl

```diff
@@ -264,6 +269,8 @@ def _accepts_flag_for_mutually_available(namer):
     _accepts_flag_for_mutually_available_test(
         name = "accepts_flag_for_mutually_available",
         target_under_test = "accepts_flag_for_mutually_available__foo",
+        # TODO: Remove once we test with Bazel 8+
+        tags = ["manual"],
```

Comment:
at least some CI jobs are 8.x+ ?

Reply:
Yeah i think I can remove the manual tags since I'm checking the feature in the tests now

Reply:
Removed all the manual tags